benchmarks/profiler_hold_resume_interruptions.rb

```diff
@@ -16,7 +16,7 @@ def create_profiler
 
   def run_benchmark
     Benchmark.ips do |x|
-      benchmark_time = VALIDATE_BENCHMARK_MODE ? { time: 0.01, warmup: 0 } : { time: 10, warmup: 2 }
+      benchmark_time = VALIDATE_BENCHMARK_MODE ? { time: 0.01, warmup: 0 } : { time: 60, warmup: 2 }
```

Comment:
Is this 60 intentional?

Reply:
It is! I pushed 01b37f8954cc30c27ba1609fb3d6851b6be7b94c as a last-ditch attempt after I saw a false performance change in this PR...

And now I'm seeing another one coming from the tracing benchmarks. I'm going to try tweaking them to 60 seconds, and if that doesn't do it, I'll throw this PR away :/